content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog
+title: 'Kubernetes v1.26: CPUManager goes GA'
+date: 2022-12-01
+slug: cpumanager-goes-GA
+---
+
+**Author:**
+Francesco Romani (Red Hat)
+
+The CPU Manager is a part of the kubelet, the Kubernetes node agent, which enables the user to allocate exclusive CPUs to containers.
+Since Kubernetes v1.10, where it [graduated to Beta](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager proved itself reliable and
+fulfilled its role of allocating exclusive CPUs to containers, so adoption has steadily grown making it a staple component of performance-critical and
+low-latency setups.  Over time, most changes were about bugfixes or internal refactoring, with the following noteworthy user-visible changes:
+
+- [support explicit reservation of CPUs](https://github.com/kubernetes/kubernetes/pull/83592): it was already possible to request to reserve a given
+  number of CPUs for system resources, including the kubelet itself, which will not be used for exclusive CPU allocation. Now it is possible to also
+  explicitly select which CPUs to reserve instead of letting the kubelet pick them up automatically.
+- [report the exclusively allocated CPUs](https://github.com/kubernetes/kubernetes/pull/97415) to containers, much like is already done for devices,
+  using the kubelet-local [podresources API](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources).
+- [optimize the usage of system resources](https://github.com/kubernetes/kubernetes/pull/101771), eliminating unnecessary sysfs changes.
+
+The CPU Manager reached the point on which it "just works", so in Kubernetes v1.26 it is graduating to GA.
+
+### Stable does not mean inflexible: customization options for CPU Manager
+
+The CPU Manager supports two operation modes, configured using its _policies_. With the `none` policy, the CPU Manager allocates CPUs to containers
+without any specific constraint except the quota set in the Pod Spec, if present. With the `static` policy, when the pod is in the Guaranteed QoS Class and
+finally when the amount of CPU requested by containers is integral the CPU Manager allocates CPUs exclusively.
```

Comment:
without any specific constraint except the (optional) quota set in the Pod spec.
With the `static` policy, then provided that the pod is in the `Guaranteed` QoS class **and**
every container in that Pod requests an integer amount of vCPU cores, then
the CPU Manager allocates CPUs exclusively. Exclusive assignment means that
other containers (whether from the same Pod, or from a different Pod) do not
get scheduled onto that CPU.
```

Reply:
thanks, rephrased as you suggested.